### PR TITLE
[AKS]Support aks egress-endpoints list operation

### PR DESCRIPTION
**Pull request by gossion**
Support aks egress-endpoints list operation

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/azext_aks_preview/_help.py [hunk 1]
```diff
@@ -1167,3 +1167,13 @@
     type: command
     short-summary: List pod identity exceptions in a managed Kubernetes cluster
 """
+
+helps['aks egress-endpoints'] = """
+    type: group
+    short-summary: see detail usage in 'az aks egress-endpoints list'.
```
Comment (zhoxing-ms) [inline src/aks-preview/azext_aks_preview/_help.py:1173]: What does this mean?

File: src/aks-preview/azext_aks_preview/_help.py [hunk 2]
```diff
@@ -1167,3 +1167,13 @@
...
+"""
+
+helps['aks egress-endpoints list'] = """
+    type: command
+    short-summary: list egress endpoints that are required or recommended to be whitelisted for a cluster.
```
[the remark this answers is not in the document]
Comment (gossion) [reply src/aks-preview/azext_aks_preview/_help.py]: fixed, thanks.

File: src/aks-preview/azext_aks_preview/_help.py (hunk 1 above)
Comment (gossion) [reply src/aks-preview/azext_aks_preview/_help.py]: changed to `short-summary: Commands to manage egress endpoints in managed Kubernetes cluster.` to get a clear summary.

**Review by miwithro — APPROVED** [review]

**Comment by gossion** [comment]
```
ERROR: Could not install packages due to an OSError: HTTPSConnectionPool(host='files.pythonhosted.org', port=443): Max retries exceeded with url: /packages/49/8d/58008a9a86075827f99aa8bb75d8db515bb9c34654f95e647cda31987db7/adal-1.2.7-py2.py3-none-any.whl (Caused by NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ff21ff51a58>: Failed to establish a new connection: [Errno 101] Network is unreachable',))
```
the build failed for network issue, @zhoxing-ms can you help to rerun the build/test?

**Review by feiskyer — APPROVED** [review]

File: src/aks-preview/azext_aks_preview/_help.py (hunk 1 above)
Comment (zhoxing-ms) [reply src/aks-preview/azext_aks_preview/_help.py]: This command group may be able to extend other sub commands later, right? Such as `aks egress-endpoints show`, `aks egress-endpoints create`...
If so, please change it to `Manage egress-endpoints`

Comment (gossion) [reply src/aks-preview/azext_aks_preview/_help.py]: I don't get you clear, do you mean `short-summary: Manage egress-endpoints`?

Comment (zhoxing-ms) [reply src/aks-preview/azext_aks_preview/_help.py]: I read your updated help and I think it's OK~

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**